cmd/experimental/kjobctl/pkg/cmd/describe/describe_printer.go

```diff
@@ -109,6 +110,10 @@ func describeJob(job *batchv1.Job) (string, error) {
 			w.Write(IndentLevelZero, "Pods Statuses:\t%d Active (%d Ready) / %d Succeeded / %d Failed\n", job.Status.Active, *job.Status.Ready, job.Status.Succeeded, job.Status.Failed)
 		}
 
+		if path, ok := job.Annotations[constants.ScriptAnnotation]; ok {
+			w.Write(IndentLevelZero, "ScriptPath:\t%s\n", path)
```

Comment:
Maybe it's a good idea to show it like Labels L89. To list all annotations. WDYT?